com.github.donadigo.appeditor

```diff
@@ -0,0 +1,81 @@
+{
+    "app-id": "com.github.donadigo.appeditor",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.donadigo.appeditor",
+    "build-options": {
+        "cflags": "-O2",
+        "cxxflags": "-O2"
```

Comment:
These shouldn't be needed, setting the build types on the projects sets compiler flags.

Reply:
Done